src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/RoleAssignmentVerificationService.java

```diff
@@ -64,9 +68,17 @@ public TaskResource verifyRoleAssignments(String taskId,
 
             if (optionalTaskResource.isEmpty()) {
                 if (customErrorMessage != null) {
+                    cftSensitiveTaskEventLogsDatabaseService.processSensitiveTaskEventLog(taskId,
```

Comment:
There is a code duplication here. You can simplify by chossing what ErrorMessage to use depneds on if customErrorMessage is null or not and then call processSensitiveTaskEventLog only once with seleceted error message.